style-guide/about-style-guide.md

```diff
@@ -0,0 +1,168 @@
+# The style guide
+
+The style guide template itself occasionally includes placeholder text that
+is intended for you to replace with your own text. This text is indicated in
+{curly brackets}.
+
+## When do I need a style guide?
+
+A style guide provides project contributors with general guidelines for writing
+project documentation. The overall goal of a style guide is to ensure quality
+and consistency throughout the project’s documentation, which is especially
+important if different authors are contributing to the documentation over time.
+
+A style guide can provide early guidance to new contributors as they make
+decisions related to your project’s documentation. Contributors often have to
+make subjective judgment calls about the documentation, such as:
+
+- Which style guide your project will defer to as its default style guide
+- How to refer to terms that are unique or specific to that project
+- How to format certain types of content
+- What tone, attitude, or level of formality you recommend using in the project
+  (usually conveyed through word choice)
+- Any specific rules you want to highlight or call out to documentation
```

Comment:
Maybe remove "or call out"? "Highlight" should be enough.

Reply:
I also like this suggestion and I've adopted it. 👍